contracts/src/v0.1/CoordinatorBase.sol

```diff
@@ -146,6 +147,31 @@ abstract contract CoordinatorBase is Ownable, ICoordinatorBase {
         return serviceFee + maxGasCost;
     }
 
+    function estimateFeeByAcc(
+        uint64 reqCount,
+        uint8 numSubmission,
+        uint32 callbackGasLimit,
+        uint64 accId,
+        IAccount.AccountType accType
+    ) public view returns (uint256) {
+        (uint256 sStartTime, uint256 sPeriod, , uint256 sSubscriptionPrice) = sPrepayment
```

Comment:
We should be careful with variable names. In this repository, `s` prefix represent state variables, however, `sStartTime`, `sPeriod` and `sSubscriptionPrice` are not state variables, but rather local variables.